utils.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+func FormatTime(microseconds uint32) string {
+	if microseconds < 1000 {
+		return fmt.Sprintf("%d µs", microseconds)
+	}
+
+	duration := time.Duration(microseconds) * time.Microsecond
+
+	var value float64
+	var unit string
+
+	switch {
+	case microseconds < 1000000:
+		value = float64(microseconds) / 1000
```

Comment:
Do you think we can instead use an approach similar to what [`countBytes`](https://github.com/checkpoint-restore/go-criu/blob/master/crit/utils.go#L46) does in go-criu?

Reply:
`countBytes()` is good but this function is simpler — it converts only `µs` to `ms` or `s`. Thus, we have fewer conditions and adding a for loop would make it more complicated.